### ISSUE TITLE
Clear history

### ISSUE THREAD

**Issue by lucerion** [Ajaxy/telegram-tt#135]
Could you, please, add Clear history functionality like in the mobile and desktop versions.

**Comment by milichev** [comment]
@lucerion, I agree, that the difference between mobile and web UX is a bit confusing, but deleting history is already implemented:

1. From the chat header menu, pick *Delete chat*
2. In the popup, click *Delete for me and %@*, where `%@` stands for the contact name.

This calls the `deleteHistory` API method.

**Comment by lucerion** [comment]
@milichev difference between Delete chat and Delete history is after clearing history chat is already in the list.
